app/models/loader/orchestrate.rb

```diff
@@ -118,6 +118,67 @@ def load
       store_passwords
 
       store_public_keys
+
+      emit_audit
+    end
+
+    SDID = Audit::SDID
+
+    OPS_MAP = {
+      "INSERT" => "add",
+      "DELETE" => "remove",
+      "UPDATE" => "change"
+    }
+
+    SUBJECT_MAP = {
+      :resource_id => 'resource'.freeze,
+      :role_id => 'role'.freeze,
+      :member_id => 'member'.freeze,
+      :name => 'annotation'.freeze,
+      :privilege => 'privilege'.freeze
+    }
+
+    def emit_audit
+      auth = policy_version.role.id
+      data = {
+        SDID::POLICY => { id: policy_id, version: policy_version.version },
+        SDID::AUTH => { user: auth }
+      }
+      format = "#{auth} %sed %s".freeze
```

Comment:
What is the significance of this particular format string?

Reply:
This is just for the human-readable message.

Reply:
Please comment accordingly.